README.rst

```diff
@@ -66,6 +66,23 @@ Example
             {'code': '6*7', 'result': '42'}
         ]
 
+        # Samples of code which needs completion
+        completion_samples = [
+            {'text': 'my_long', 'matches': ['my_long_variable_name', 'my_long_function_name']},
+            {'text': 'a.', 'matches': ['a.foo', 'a.bar']}
```

Comment:
Is it worth copying the example from `test_ipykernel.py` here? I'm looking at these examples and wondering where the setup code is to create the variables you're completing.

Reply:
You're right, those examples are not relevant. We could use setup code to create some variables, or use built-in functions? I'll take a look at `test_ipykernel.py`

Reply:
Actually I didn't see you had examples on the repo. Maybe we could make it clearer that it's examples? And add a link to those examples in the README?

Reply:
Yup, by all means. Maybe keep a shortened example in the README itself so it's readable at a glance.